climpred_skeleton/comparison.py

```diff
@@ -0,0 +1,45 @@
+from abc import ABC, abstractmethod
+
+import xarray as xr
+
+
+class Comparison(ABC):
+    def __init__(self, prediction: xr.Dataset, observation: xr.Dataset):
+        self.prediction = prediction
+        self.observation = observation
+        self.nmember = self.prediction['member'].size
+        self.members = self.prediction['member'].data
+
+    def factory(self, type):
+        if type == 'e2o':  # Convert to dictionary with all aliases.
```

Comment:
I need to think about how to construct the aliases smartly here. But this will definitely be a dictionary, e.g. 

```python
{
'e2o': e2oComparison,
'm2o': m2oComparison,
}
```